src/CoreShop/Bundle/CoreBundle/EventListener/CartBlamerListener.php

```diff
@@ -116,6 +117,11 @@ private function blame(CustomerInterface $user)
      */
     private function getCart()
     {
-        return $this->cartContext->getCart();
+        try {
+            return $this->cartContext->getCart();
+        }
+        catch (CartNotFoundException $ex) {
+            return null;
```

Comment:
The `@return` PhpDoc should be adapted to `@return CartInterface|null`

Reply:
yep, thats what stan is telling as well :) Thanks though :)